### PR TITLE
Clickhouse/profiler

### PR DESCRIPTION
**Pull request by ykobuk**
Add the Clickhouse profiler.
Screenshot from 2023-03-31 17-32-40
Screenshot from 2023-03-31 17-34-05

### CODE REVIEW

File: odd_collector_profiler/domain/config.py [hunk 1]
```diff
@@ -91,3 +91,19 @@ def connection_str(self) -> str:
...
     type = "azure_sql"
+
+
+class ClickHouseConfig(DatabaseConfig):
+    type = "clickhouse"
+    database: str
+    host: str
+    port: str
+    username: str
+    password: str
```
Comment (Vixtir) [inline odd_collector_profiler/domain/config.py]: Use `SecretStr` from pydantic

File: odd_collector_profiler/profilers/clickhouse/profiler.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+from oddrn_generator import ClickHouseGenerator
+
+from odd_collector_profiler.datasource.database.profiler import RDBProfiler
+from odd_collector_profiler.datasource.database.repository import RDBRepository
+from odd_collector_profiler.domain.config import ClickHouseConfig
+from odd_collector_profiler.domain.profiler import Profiler
+from odd_collector_profiler.profilers import DATA_PROFILER
+
+SYS_SCHEMAS = {
```
Comment (Vixtir) [inline odd_collector_profiler/profilers/clickhouse/profiler.py]: Does Clickhouse have that schemas?

File: odd_collector_profiler/domain/config.py [hunk 3]
```diff
@@ -91,3 +91,19 @@ def connection_str(self) -> str:
...
+    secure: bool = False
```
Comment (Vixtir) [inline odd_collector_profiler/domain/config.py]: Remove if `secure` `verify` and `query_limit` aren't used by now.

**Review by Vixtir — CHANGES_REQUESTED** [review]

File: odd_collector_profiler/profilers/clickhouse/profiler.py (hunk 2 above)
Comment (ykobuk) [reply odd_collector_profiler/profilers/clickhouse/profiler.py]: The same as MsSQL

File: odd_collector_profiler/domain/config.py (hunk 3 above)
Comment (ykobuk) [reply odd_collector_profiler/domain/config.py]: Removed

File: odd_collector_profiler/domain/config.py (hunk 1 above)
Comment (ykobuk) [reply odd_collector_profiler/domain/config.py]: Done.

**Review by Vixtir — APPROVED** [review]

**Pull request merged**